install.d/51-dracut-rescue.install

```diff
@@ -3,15 +3,16 @@
 export LANG=C
 
 COMMAND="${1:?}"
-KERNEL_VERSION="${2:?}"
-BOOT_DIR_ABS="${3%/*}/0-rescue"
-KERNEL_IMAGE="${4:?}"
 
 # If the initrd was provided on the kernel command line, we shouldn't generate our own.
 if [[ "$COMMAND" = "add" && "$#" -gt 4 ]]; then
     exit 0
 fi
 
+KERNEL_VERSION="${2:?}"
+BOOT_DIR_ABS="${3%/*}/0-rescue"
+KERNEL_IMAGE="${4:?}"
```

Comment:
This won't help here because this script also runs on `kernel-install remove` (unlike `50-dracut.install`, note that the if statement above is slightly different)

I suggest to instead simply remove the `:?` on `KERNEL_IMAGE` which is the one that is not going to be set when we call remove. Systemd's `90-loadentry.install` has this, so doing the same here should be fine:
```
COMMAND="${1:?}"
KERNEL_VERSION="${2:?}"
ENTRY_DIR_ABS="${3:?}"
KERNEL_IMAGE="$4"
```

And then for symmetry reasons I would suggest to also fix the problem in this same way in `50-dracut.install`

Reply:
Thanks! I overlooked that `51-dracut-rescue.install` had a slightly different `if` statement. I thought I tested it, but I forgot I had symlinked `/etc/kernel/install.d/51-dracut-rescue.install` to `/dev/null` (whoops).

This should be fixed now.